source/_includes/common-tasks/data_disk.md

```diff
@@ -28,7 +28,7 @@ The data disk feature can be used on an existing installation without losing dat
 
 1. Connect the data disk to your system.
 2. Go to **{% my storage title="Settings > System > Storage" %}** in the UI.
-3. Press the three dots on the top right and choose "Move datadisk"
+3. Click the "Move data disk" button
```

Comment:
```suggestion
3. Select the "Move data disk" button.
```
I know click is frequently used throughout the docs. But select is the preferred term, because click is more specific to [mouse interaction](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click)